modules/framework-cats/shared/src/test/scala/DogFoodTests.scala

```diff
@@ -300,6 +300,105 @@ object DogFoodTests extends IOSuite {
     }
   }
 
+  test(
+    "expect.same delegates to default multi-line show when no instance is found") {
+    _.runSuite(Meta.Rendering).flatMap {
+      case (logs, _) =>
+        val actual =
+          extractFailureMessageForTest(logs, "(same with default show)")
+        if (ScalaCompat.isScala2_12) {
+          assertInlineSnapshot(
+            actual,
+            """- (same with default show) 0ms
+  Values not equal: (src/main/DogFoodTests.scala:5)
+
+  => Diff (- obtained, + expected)
```

Comment:
Scala 2.12 doesn't support field names, hence this test is different for different Scala versions.